elixir-sdk/native/sdk_core/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sdk_core"
+version = "0.1.0"
+authors = []
+edition = "2021"
+
+[lib]
+name = "sdk_core"
+crate-type = ["cdylib"]
+
+[dependencies]
+rustler = "0.36.1"
+eppo_core = { path = "../../../eppo_core", features = ["rustler"] }
```

Comment:
this needs an actual dependency, you can override like this for local testing: https://github.com/Eppo-exp/eppo-multiplatform/pull/197#issuecomment-2680522104

just bumped `eppo_core` to `9.0.0`